rasa_nlu/server.py

```diff
@@ -214,6 +214,23 @@ def train(self, request):
             request.setResponseCode(500)
             returnValue(json_to_string({"error": "{}".format(e)}))
 
+    @app.route("/evaluate", methods=['POST'])
+    @requires_auth
+    @check_cors
+    def evaluate(self, request):
+        data_string = request.content.read().decode('utf-8', 'strict')
+        kwargs = {key.decode('utf-8', 'strict'): value[0].decode('utf-8', 'strict')
+                  for key, value in request.args.items()}
+        request.setHeader('Content-Type', 'application/json')
+
+        try:
+            request.setResponseCode(200)
+            response = self.data_router.evaluate(data_string, kwargs)
```

Comment:
either `**kwargs` or parse the arguments separately (I'd favor the second way, as it makes clear which arguments this endpoint accepts)